.ansible-lint

```diff
@@ -4,5 +4,6 @@ skip_list:
    - '303'  # systemctl used in place of systemd module
    - '701'  # Role info should contain platforms
 warn_list:
+   - '204'  # Lines should be no longer than 160 chars
```

Comment:
They aren't going to go for this in tripleo-ansible, I spent all morning shortening my lines. You must pass their tox tests to be accepted and that includes yamllint and ansible-lint without our own exception rules.

https://github.com/openstack/tripleo-ansible/blob/master/.ansible-lint
https://github.com/openstack/tripleo-ansible/blob/master/tox.ini